node/pkg/allocateip/allocateip_test.go

```diff
@@ -719,8 +916,14 @@ var _ = Describe("Running as daemon", func() {
 
 		// Create client and IPPool (IPIP)
 		c, _ = client.New(*cfg)
-		pool, err = c.IPPools().Create(ctx, makeIPv4Pool("pool1", "172.16.0.0/26", 26), options.SetOptions{})
+
+		poolV4, err = c.IPPools().Create(ctx, makeIPPool("pool1-v4", "172.16.0.0/26", 26, api.IPIPModeAlways, api.VXLANModeNever), options.SetOptions{})
 		Expect(err).ToNot(HaveOccurred())
+
+		// Create /128 IPv6 IPPool so that there is only one address to be assigned
```

Comment:
Yeah, I think we should probably just implement the validator changes in this PR - not a fan of adding commented out code with the expectation of uncommenting it later. 

Should either add the validator changes in this PR, or add these test assertions in a future PR.

Reply:
I still think the validation changes should not go in at this moment, so I could remove the commented out test stuff and push those in with the validation changes? How does that sound?

Reply:
After an offline sync, we agreed to just remove the libcalico-go IPv6 VXLAN validation in this PR and uncomment the test code. Thanks, Casey!